### PR TITLE
doc: rework example scripts to standard bash

### PR DESCRIPTION
**Pull request by rubensfig**

### CODE REVIEW

File: examples/routing/VRFs/01-controller [hunk 1]
```diff
@@ -95,28 +94,36 @@ function setup() {
...
+  ip link add link ${BRIDGE} name ${BRIDGE}.${BR_VLAN3} type vlan id ${BR_VLAN3}
+  bridge vlan add vid ${BR_VLAN3} dev ${BRIDGE} self
+  ip link set ${BRIDGE}.${BR_VLAN3} up
 
   ## SVI - VRF enslavement
-  for i in {1..${#BR_VLAN_LIST[@]}}; do
-    ip link set ${BRIDGE}.${BR_VLAN_LIST[$i]} vrf ${VRF_LIST[$i]}
-  done
+  ip link set ${BRIDGE}.${BR_VLAN} vrf ${VRF}
+  ip link set ${BRIDGE}.${BR2_VLAN} vrf ${VRF2}
```
Comment (DFritzsche) [inline examples/routing/VRFs/01-controller]: use `BR_VLAN2` and `BR_VLAN3`

File: examples/routing/VRFs/01-controller [hunk 2]
```diff
@@ -46,30 +46,29 @@ BRIDGE=${BRIDGE:-swbridge}
...
-SVI_IP_LIST=($SVI_IP $SVI_IP2 $SVI_IP3)
-NEI_IP=${NEI_IP:-10.0.100.2}
 
 function setup() {
   ## vrf
-  for i in {1..${#VRF_LIST[@]}}; do
-    ip link add ${VRF_LIST[$i]} type vrf table ${VRF_ID_LIST[$i]}
-    ip link set ${VRF_LIST[$i]} up
-  done
+  ip link add ${VRF} type vrf table ${VRF_}
```
Comment (DFritzsche) [inline examples/routing/VRFs/01-controller]: use `VRF_TABLE_ID`

File: examples/bridging/01-controller [hunk 3]
```diff
@@ -15,28 +15,30 @@
...
-# vids
-for vid in $(eval echo $BR_VLAN_RANGE) ; do
-  bridge vlan add vid ${vid} dev ${PORTA}
-  bridge vlan add vid ${vid} dev ${PORTB}
-done
+  # vids
+  for vid in $(seq 2 $BR_VLAN_RANGE) ; do
+    bridge vlan add vid ${vid} dev ${PORTA}
+    bridge vlan add vid ${vid} dev ${PORTB}
+    bridge vlan add vid ${vid} dev ${swbridge} self
```
Comment (DFritzsche) [inline examples/bridging/01-controller]: replace `swbridge` with `BRIDGE` and this works, although I am no sure if adding vids to the swbridge is really required

**Comment by hilmarm** [comment]
Remember to add a VRF section to the examples/README.MD

**Pull request merged**